### PR TITLE
Add some integration tests for starlark-repl

### PR DESCRIPTION
**Pull request by illicitonion**
Starts on #38

### CODE REVIEW

File: starlark-repl/tests/integration.rs [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    let f1 = make_file("x = 1");
+    let f2 = make_file("x");
+
+    Command::main_binary()
+        .unwrap()
+        .arg(f1.path())
+        .arg(f2.path())
+        .assert()
+        .code(2)
+        .stderr(contains("Variable 'x' not found"));
```
Comment (damienmg) [inline starlark-repl/tests/integration.rs]: Assert on the error code rather. The message can change, the error code should not.

**Comment by damienmg** [comment]
I sent you an invite to give you write access to this repository :)

File: starlark-repl/tests/integration.rs (hunk 1 above)
Comment (illicitonion) [reply starlark-repl/tests/integration.rs]: Done

**Comment by illicitonion** [comment]
> I sent you an invite to give you write access to this repository :)

Thank you very much! I'm sure we'll have fun trading reviews with each other :D

**Pull request merged**